dist/routable-component.js

```diff
@@ -100,20 +100,20 @@ var routableComponent =
 	   * constructor
 	   * @param {Object<EventEmitter>} [components] コンポーネントの連想配列
 	   * @param {RoutableComponentRoutes} routes ルーティング
-	   * @param {Object<RoutableComponentController>} controller_classes コントローラクラスの連想配列
+	   * @param {Object<RoutableComponentController>} controllerClasses コントローラクラスの連想配列
 	   */
 	
-	  function RoutableComponent(components, routes, controller_classes) {
+	  function RoutableComponent(components, routes, controllerClasses) {
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)
* Error - Missing JSDoc comment. (require-jsdoc)
* Warning - 'RoutableComponent' is already declared in the upper scope. (no-shadow)

<!-- Comment by [SideCI](https://sideci.com) -->